src/components/Modal.jsx

```diff
@@ -195,6 +248,8 @@ Modal.defaultProps = {
   title: 'Please give me a title :)',
   content: 'Please give me some content :)',
   image: '',
+  sound: '',
```

Comment:
Yep, that makes sense, and if no sound property is specified, it'll default to the value you set here. If that value is just `false`, it'll make checking for the existence of the sound a bit easier :) I should probably do that for image as well. I _think_ that will work..